### ISSUE TITLE
[BUG] Issue with Go 1.16

### ISSUE THREAD

**Issue by alexjomin** [unidoc/unipdf#437]
## Description
As I am on a Macbook M1, I have the Go version 1.16 installed and I can't get unipdf compiles with Go 1.16 (works with 1.15 on another Mac)

## Actual Behavior
1. Install Go 1.16 beta 
2. Copy one of unipdf examples
3. `go run main.go`
4. See error:

```
github.com/unidoc/unipdf/v3/model
<autogenerated>:1: internal compiler error: child dcl collision on symbol _febf within "".(*PdfAppender).replaceObject"
```

**Comment by ahall** [comment]
Was this solved @alexjomin ?

**Comment by moogacs** [comment]
facing the same bug , @alexjomin is it fixed ?

**Comment by alexjomin** [comment]
AFAIK It is not, 

no answer so I moved to another library.

**Comment by gunnsth** [comment]
Reopening as this is not resolved.

**Comment by ahall** [comment]
@alexjomin Go 1.16 is released now so top priority for us to fix this, sorry about this.

**Comment by aubelsb2** [comment]
Got this on 
```
>go version
go version go1.16 windows/amd64
```

**Comment by Kleissner** [comment]
Same issue here after upgrading to Go 1.16

**Comment by peterwilliams97** [comment]
I can't reproduce this on my Macbook, I have tried a few of the examples.
```
 % go version
go version go1.16 darwin/amd64
```
Which example should I be running?

**Comment by diegomontoya** [comment]
@peterwilliams97  Arch Linux x86_64 3.12/3.13 all reproduce this compile bug.

**Comment by diegomontoya** [comment]
@peterwilliams97 For us the following code triggered compile bug on go 1.16

```
pdfReader, err := model.NewPdfReader(bytes.NewReader(byteSlice))
```

**Comment by peterwilliams97** [comment]
Thanks @diegomontoya. I was using some development code. I am getting the bug now,

**Comment by Kunamatata** [comment]
Just wanted to also say that I am experiencing this after updating to go 1.16
```➜ go version
go version go1.16 darwin/amd64```

**Comment by ahall** [comment]
This is resolved in the new version: v3.19.1.
